### DISCUSSION TITLE
/ in database name (connection url)

### DISCUSSION THREAD

**Discussion by kdev** [prisma/prisma#10468] [Q&A]
When trying to create a database named something like this: `org/<name>`  it seems that it's not possible to do this using a connection url. Prisma is always generating a database named `org` and not `org/<name>`.

Any ideas on how to accomplish that? 

(I'm using PostgreSQL and know that it's possible to have `/` in database names)

**Comment by pantharshit00** [comment]
You will need to use percent encoding here. `/` in percent encoding is `%2F` so replace `/` with `%2F` in your URI.

**Reply by janpio** [reply]
That is _just_ creates `org` is a bit unexpected anyway though I would say.

**Reply by kdev** [reply]
> You will need to use percent encoding here. `/` in percent encoding is `%2F` so replace `/` with `%2F` in your URI.

Thats not working, it literally creates a database named `org%2f<name>`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
